msmbuilder/featurizer/multichain.py

```diff
@@ -0,0 +1,510 @@
+# Author: Brooke Husic <brookehusic@gmail.com>
+# Contributors: 
+# Copyright (c) 2016, Stanford University and the Authors
+# All rights reserved.
+
+from __future__ import print_function, division, absolute_import
+
+import warnings
+
+import mdtraj as md
+from mdtraj.utils import ensure_type
+from mdtraj.utils.six import string_types
+from mdtraj.utils.six.moves import xrange
```

Comment:
I pretty much moved over the [`mdtraj` code](https://github.com/mdtraj/mdtraj/blob/3aecf322b8555dc0f941382256be75f86459d575/mdtraj/geometry/contact.py#L32) exactly as it was. Should I update it?